util/execdetails/execdetails.go

```diff
@@ -40,19 +40,26 @@ var (
 
 // ExecDetails contains execution detail information.
 type ExecDetails struct {
-	CalleeAddress    string
-	CopTime          time.Duration
-	ProcessTime      time.Duration
-	WaitTime         time.Duration
-	BackoffTime      time.Duration
-	LockKeysDuration time.Duration
-	BackoffSleep     map[string]time.Duration
-	BackoffTimes     map[string]int
-	RequestCount     int
-	TotalKeys        int64
-	ProcessedKeys    int64
-	CommitDetail     *CommitDetails
-	LockKeysDetail   *LockKeysDetails
+	CalleeAddress             string
+	CopTime                   time.Duration
+	ProcessTime               time.Duration
+	WaitTime                  time.Duration
+	BackoffTime               time.Duration
+	LockKeysDuration          time.Duration
+	BackoffSleep              map[string]time.Duration
+	BackoffTimes              map[string]int
+	RequestCount              int
+	TotalKeys                 int64
+	ProcessedKeys             int64
+	CommitDetail              *CommitDetails
+	LockKeysDetail            *LockKeysDetails
+	ProcessedVersions         uint64
+	TotalVersions             uint64
+	RocksdbDeleteSkippedCount uint64
+	RocksdbKeySkippedCount    uint64
+	RocksdbBlockCacheHitCount uint64
+	RocksdbBlockReadCount     uint64
+	RocksdbBlockReadBytes     uint64
```

Comment:
How about use a new struct such as `CopDetail` to contain those runtime information, because not every statement contains cop request, such as insert.